config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   get 'home', to: 'home#index'
+  post 'cards', to: 'cards#compare'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.